README.md

```diff
@@ -15,10 +15,46 @@ adds the following new functionality:
   abstractions](https://crossplane.io/docs/master/concepts.html), enabling
   Alibaba Cloud resources to fulfill a user's general need for cloud services
 
-## Getting Started and Documentation
+## Getting Started
 
-For getting started guides, installation, deployment, and administration, see
-our [Documentation](https://crossplane.io/docs/latest).
+In the following, We will deploy flight tracker app using OAM and Alibaba RDS resource:
+
+![alt text](./doc/img/flight-tracker.png)
+
+Make sure you have installed:
+
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- kubectl
+- [helm](https://helm.sh/)
+
+Then set up the environment on first trial:
+
+```bash
+helm repo add crossplane-master https://charts.crossplane.io/master/
+git clone git@github.com:crossplane/addon-oam-kubernetes-local.git ../addon-oam-kubernetes-local
+export ACCESS_KEY_ID=<your-access-key-id>
+export ACCESS_KEY_SECRET=<your-access-key-secret>
+```
+
+Access key should be obtained via Alibaba clooud.
+
+Then run command:
+```bash
+make demo
+```
+This will deploy the [Crossplane](https://github.com/crossplane/crossplane), [addon-oam-kubernetes-local](https://github.com/crossplane/addon-oam-kubernetes-local), setting up cloud provider, and finally deploy an OAM app with a cloud RDS database.
```

Comment:
Users will also be interested about:

1. How we enable provider by  `./cluster/examples/provider.yaml`
2. Why we need declare `./cluster/examples/database/resource_class.yaml`
3. How finally resource is created by  `./hack/demo/deploy/`

Reply:
That's more like a explanation doc rather than a getting started section. Usually in getting started section it will just provide some commands for users to copy&paste.

Reply:
fine, maybe we could create another issue for that